bundle.yaml.j2

```diff
@@ -0,0 +1,61 @@
+---
+bundle: kubernetes
+name: tempo-bundle
+description: >
+  Tempo is a distributed tracing backend by Grafana, supporting Jaeger, Zipkin, and OpenTelemetry protocols.
+
+website: https://github.com/canonical/tempo-bundle
+issues: https://github.com/canonical/tempo-bundle/issues
+
+applications:
+  tempo:
+    charm: tempo-coordinator-k8s
+    channel: {{ channel|default('edge', true) }}
+    scale: 1
+    trust: true
+    constraints: arch=amd64
```

Comment:
You'll also need `resources: nginx-image: (...) nginx-prometheus-exporter-image:` as you can see here: https://github.com/canonical/tempo-coordinator-k8s-operator/pull/4/files#diff-8af267359854d30b091fde763088e458bf27c8fedc056895f3104ef05b61f4b5R25